test/integration/pullrequests.test.js

```diff
@@ -0,0 +1,139 @@
+const chai = require('chai')
+const sinon = require('sinon')
+const { expect } = chai
+const chaiHttp = require('chai-http')
+
+const githubService = require('../../services/githubService')
+const userModel = require('../../models/users')
+const cleanDb = require('../utils/cleanDb')
+
+const app = require('../../server')
+
+chai.use(chaiHttp)
+
+// Import fixtures
+const githubPRInfo = require('../fixtures/contributions/githubPRInfo')()
+const { getOpenPRs, getStalePRs, pullRequestKeys } = require('../fixtures/pullrequests/pullrequests')
+const openPRs = getOpenPRs()
+const stalePRs = getStalePRs()
+const { data: { items } } = openPRs
+const earliestDate = items[0].created_at
+const oldestDate = items[items.length - 1].created_at
+
+describe('Pull Requests', function () {
+  const username = 'prakash'
```

Comment:
Can we simply utilise the existing user fixtures, or add this one to the fixtures array?